src/view/com/util/moderation/PostHider.tsx

```diff
@@ -70,7 +67,14 @@ export function PostHider({
 
   // NOTE: any further label enforcement should occur in ContentContainer
   return (
-    <Link testID={testID} style={style} href={href} noFeedback>
+    <Link
+      testID={testID}
+      style={style}
+      href={href}
+      noFeedback
+      accessible={true}
+      accessibilityRole="none"
```

Comment:
@hs4man21 FYR the main reason why actions weren't usable before was because we defaulted to `accessibilityRole="link"`, which actually prevents usage of `onAccessibilityActions`.